0x30-0x3F.md

```diff
@@ -14,6 +14,7 @@
 - DHT20 Humidity/Temperature Sensor (0x38 only)
 - [VEML6070 UV Index](https://www.adafruit.com/product/2899) (0x38 and 0x39)
 - [FT6x06 Capacitive Touch Driver](https://www.adafruit.com/product/1947) (0x38 only)
+- [NCP5623 RGB LED Driver](https://www.onsemi.com/products/power-management/led-drivers/linear-led-drivers/ncp5623d)
```

Comment:
You linked to the NCP5623D, though it does share the same datasheet as the NCP5623.  Did you want to change the link to the listed driver?  That would be:

```suggestion
- [NCP5623 RGB LED Driver](https://www.onsemi.com/products/power-management/led-drivers/linear-led-drivers/ncp5623)
```